service/topology_coordinator.cc

```diff
@@ -1567,6 +1595,9 @@ class topology_coordinator : public endpoint_lifecycle_subscriber {
                             }
                             dst = dst_opt.value().host;
                         }
+                        // we repair only the tablet of the base table, and not the tablets of other co-located tables,
```

Comment:
Please explain. Why is it ok to stop repairing some views? Where are they repaired? Why did we repair them before your change?